requirements/prod.txt

```diff
@@ -4,100 +4,85 @@
 #
 #    pip-compile --output-file=requirements/prod.txt requirements/prod.in
 #
-aioredis==1.3.1
-    # via channels-redis
 appdirs==1.4.4
     # via
     #   cumulusci
     #   fs
-asgiref==3.5.2
+asgiref==3.7.2
     # via
     #   channels
     #   channels-redis
-    #   daphne
     #   django
-async-timeout==4.0.2
-    # via
-    #   aioredis
-    #   redis
-attrs==21.4.0
+async-timeout==4.0.3
+    # via redis
+attrs==23.1.0
     # via
-    #   automat
     #   jsonschema
+    #   referencing
     #   service-identity
-    #   twisted
-authlib==1.2.0
+authlib==1.2.1
     # via
     #   cumulusci
     #   simple-salesforce
-autobahn==22.6.1
-    # via
-    #   -r requirements/prod.in
-    #   daphne
-automat==20.2.0
-    # via twisted
-beautifulsoup4==4.11.1
+autobahn==23.6.2
     # via -r requirements/prod.in
-bleach==5.0.1
+beautifulsoup4==4.12.2
+    # via -r requirements/prod.in
+bleach==6.1.0
     # via
     #   -r requirements/prod.in
     #   sfdo-template-helpers
-certifi==2022.12.7
+certifi==2023.7.22
     # via
     #   cumulusci
     #   requests
     #   sentry-sdk
     #   snowfakery
-cffi==1.15.1
+cffi==1.16.0
     # via
     #   cryptography
     #   cumulusci
-channels==3.0.5
+channels==4.0.0
     # via
     #   -r requirements/prod.in
     #   channels-redis
-channels-redis==3.4.1
+channels-redis==4.1.0
     # via -r requirements/prod.in
-charset-normalizer==3.0.1
+charset-normalizer==3.2.0
     # via
     #   cumulusci
     #   requests
     #   snowfakery
-click==8.1.3
+click==8.1.6
     # via
     #   cumulusci
     #   rq
     #   snowfakery
-constantly==15.1.0
-    # via twisted
-croniter==1.3.5
+crontab==1.0.1
     # via rq-scheduler
-cryptography==39.0.1
+cryptography==41.0.7
     # via
     #   authlib
     #   autobahn
     #   cumulusci
+    #   django-anymail
     #   pyjwt
-    #   pyopenssl
     #   secretstorage
     #   service-identity
     #   sfdo-template-helpers
-cumulusci==3.76.0
+cumulusci==3.83.0
     # via -r requirements/prod.in
-daphne==3.0.2
-    # via channels
 defusedxml==0.7.1
     # via
     #   cumulusci
     #   python3-openid
-deprecated==1.2.13
-    # via redis
-dj-database-url==0.5.0
+dj-database-url==2.1.0
     # via -r requirements/prod.in
-django==4.0.6
+django==5.0
```

Comment:
Let's stick to the LTS for now. You can restrict which version of Django we use in `prod.in`:

```
Django~=4.2.0 # LTS
```

(Where `~=` means [compatible release](https://peps.python.org/pep-0440/#compatible-release).)